addons/onboarding/models/onboarding_progress.py

```diff
@@ -0,0 +1,64 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import api, fields, models
+
+
+ONBOARDING_PROGRESS_STATES = [
+    ('not_done', 'Not done'),
+    ('just_done', 'Just done'),
+    ('done', 'Done'),
+]
+
+
+class OnboardingProgress(models.Model):
+    _name = 'onboarding.progress'
+    _description = 'Onboarding Progress Tracker'
```

Comment:
Would require a ``_rec_name`` as you don't have any name field (probably onboarding_id ? and/or define a name_get ?)